content/en/observability_pipelines/sources/http_server.md

```diff
@@ -15,4 +17,42 @@ Select and set up this source when you [set up a pipeline][1]. The information b
 
 {{% observability_pipelines/source_settings/http_server %}}
 
-[1]: /observability_pipelines/set_up_pipelines/
\ No newline at end of file
+## Send AWS vended logs with the Datadog Lambda Forwarder to Observability Pipelines
+
+To send AWS vended logs to Observability Pipelines with the HTTP/S Server source:
+
+- [Set up a pipeline with the HTTP/S Server source](#set-up-a-pipeline).
+- [Deploy the Datadog Forwarder](#deploy-the-datadog-lambda-forwarder).
+
+**Note**: This is available for Worker versions 2.51 or later.
+
+### Set up a pipeline
+
+1. Navigate to [Observability Pipelines][1].
+1. Select the HTTP Server source:
+    1. Select **None** in the dropdown menu for **Authorization strategy**.
+    1. Select **JSON** in the dropdown menu for **Decoding**.
+1. After you set up your [destinations][2] and [processors][3], click **Next: Install**.
+1. On the **Install** page, enter the HTTP/S Server listener address, such as `0.0.0.0:9997`.
+1. Follow the rest of the instructions on the page to install the Worker based on your platform.
+
+### Deploy the Datadog Lambda Forwarder
+
+Follow the [Datadog Forwarder CloudFormation installation instructions][4] to deploy the Datadog Forwarder. After you have filled in `DdApiKey` and selected the appropriate `DdSite`, go to the **Log Forwarding (Optional)** section. Enter the following information in that section to set up the Forwarder to send logs to Observability Pipelines:
+
+1. In the **DdUrl** field, enter your load balancer address. Datadog recommends fronting your Worker with a load balancer.
+1. In the **DdPort** field, enter `80`.
+1. If you want to encrypt traffic from the Datadog Forwarder to your Observability Pipelines Worker load balancer, you need to create an HTTPS listener and manage a certificate on your load balancer. See [Create an HTTPS listener for your Application Load Balancer][5] for more information.
```

Comment:
I know this is pedantic, but it is possible they don't setup their OPW in AWS, so perhaps when we reference the AWS docs we should say "For example in AWS...consult your cloud provider's documentation" or something that to that tune.